### DISCUSSION TITLE
I want the old C++ plugin back

### DISCUSSION THREAD

**Discussion by ghost** [apache/netbeans#4574] [Polls]
On the first days of Apache Netbeans there is a roadmap. Now that roadmap is no longer available (via Google search) and the old C++ plugin is also nowhere seen. I want to know when will it's back. Exactly when.

**Comment by Chris2011** [comment]
As pointed here: #4579 the old one will not come back, probably never but the new cpplite will fit in here. Step by step and I know you just want to be a consumer but you can help here anyway by doing research, help with tickets, etc.
